migrator/migrations/m_188_to_m_189_logimbues_add_time_column/test/schema/convert_log_imbues.go

```diff
@@ -0,0 +1,29 @@
+// Code originally generated by pg-bindings generator.
```

Comment:
Is this the same as migrator/migrations/m_188_to_m_189_logimbues_add_time_column/schema/convert_log_imbues.go can you reuse it?

Reply:
@c-du No, this one is to populate the old schema for the tests.  This one does not translate the time whereas the other does convert the time field for the new schema.  I'm sure I could probably figure out a way to do something with this but for one time test code, does not seem worth it.

Reply:
@c-du No, this one is to populate the old schema for the tests.  This one does not translate the time whereas the other does convert the time field for the new schema.  I'm sure I could probably figure out a way to do something with this but for one time test code, does not seem worth it.